server/api.js

```diff
@@ -13,7 +13,7 @@ router.get("/bookings", async (req, res) => {
 		);
 		// convert array with objects having {username, booking_date} into {name,date}
 		const bookings = result.rows.map((row) => {
-			return { id: row.id, name: row.username, date: row.booking_date };
+			return { id: row.id, name: row.username, table: row.desk_table, date: row.booking_date };
```

Comment:
the query should join against desk, and we should use both the desk_id and the deskname (as per the trello ticket)